src/cli/util.js

```diff
@@ -1010,7 +1010,7 @@ function updateContextArgv(context, plugins, pluginSearchDirs) {
   const argv = minimist(context.args, minimistOptions);
 
   context.argv = argv;
-  context.filePatterns = argv._;
+  context.filePatterns = argv._.map((file) => String(file));
```

Comment:
Is this the same behavior?
```suggestion
  context.filePatterns = argv._.map(String);
```

Reply:
Yes, but normally, I follow [unicorn/no-fn-reference-in-iterato](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-fn-reference-in-iterator.md) rule.

Reply:
Is the rule enable on this project? I don't have strong opinion for this coding style, but I think the rules should be enabled for uniformity.

Reply:
I can do this in another PR.